react-main/src/app/screens/Game/components/Moves/index.js

```diff
@@ -22,4 +23,11 @@ class Moves extends Component {
   }
 }
 
+Moves.propTypes = {
+  history: PropTypes.arrayOf(PropTypes.string),
```

Comment:
You are doing `history.map()` above. What if history was null?  Map is an Array method. (It can be either required or not depending on your decision, but in both cases you must add something to this component)